NETWORK-POLICIES.md

```diff
@@ -0,0 +1,22 @@
+# Test network policies
```

Comment:
I created a small README here @norbertgruszka that shows how one could test the effectiveness of the standard Kubernetes network policies. I spent quite some time in fine-tuning these policies :) Here we could also add more information on how to test the Cilium policies later on